.github/styles/Vocab/Base/accept.txt

```diff
@@ -153,6 +160,7 @@ upserted
 uuid
 UUID
 Vercel
+versioningTemplate
```

Comment:
I think we should be able to remove these exceptions if we place the words inside backticks in the post